### PR TITLE
[4.12] manifest-rhel-8.6: Update repo for kernel package

### PR DESCRIPTION
**Pull request by gursewak1997**

### CODE REVIEW

**Review by cgwalters — APPROVED** [review]

File: manifest-rhel-8.6.yaml [hunk 1]
```diff
@@ -157,7 +157,7 @@ packages:
 # Packages pinned to specific repos in RHCOS
 repo-packages:
   # we always want the kernel from BaseOS
```
Comment (cgwalters) [inline manifest-rhel-8.6.yaml:159]: This comment is out of date now, and should probably be changed to reference the reason why we're doing this.

**Comment by travier** [comment]
Don't we want this just for the RT kernel?

**Comment by ashcrow** [comment]
We want it for both vanilla and rt.

**Comment by travier** [comment]
Will need a bug, etc.

**Comment by ashcrow** [comment]
I've overridden all but the `looks good to me` label.

**Comment by ashcrow** [comment]
cc @sosiouxme

File: manifest-rhel-8.6.yaml (hunk 1 above)
Comment (gursewak1997) [reply manifest-rhel-8.6.yaml:159]: Should've updated this before re-testing but we will be switching back to BaseOS in the next few weeks.
Not ideal but let's ignore this due to time constraints.

**Pull request merged**